addons/mail/static/src/model/model_field.js

```diff
@@ -861,14 +856,12 @@ class ModelField {
             const prevLocalIds = this.read(record);
 
             // other record already unlinked, avoid useless processing
-            if (!prevLocalIds.includes(otherLocalId)) {
+            if (!prevLocalIds.delete(otherLocalId)) {
```

Comment:
shouldn't this be `.has()` instead of `.delete()`?

Reply:
> **Set.prototype.delete(value)**
>     Removes the element associated to the value and returns the value that Set.prototype.has(value) would have previously returned. Set.prototype.has(value) will return false afterwards.

This is just to do it in one step instead of 2.

Reply:
ok